packages/context/src/index.ts

```diff
@@ -103,7 +104,15 @@ export class Services implements ServicesInterface {
       idbConstants: indexedDb.constants(),
     });
 
+    // Dynamically fetch binary file for genesis compact block
```

Comment:
fetch api goes through the browser's network stack for local files, and there seems to be overhead in setting up the request, but latency is minimal